rest-service/manager_rest/storage/resource_models.py

```diff
@@ -5,6 +5,7 @@
 from os import path
 from datetime import datetime
 from collections import namedtuple
+from sqlalchemy import and_ as sql_and
```

Comment:
no need to import it:
1. `db.and_` (ditto `db.or_`)
2. just multiple `.filter().filter()` calls are ANDed anyway